lang/cs/Org.Apache.REEF.Network.Tests/GroupCommunication/GroupCommunicationTests.cs

```diff
@@ -380,6 +380,45 @@ public void TestBroadcastOperator()
             Assert.Equal(value, receiver2.Receive());
         }
 
```

Comment:
In GroupCommunicationTreeTopologyTests.TestScatterReduceOperatorsWithCancelation